maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -141,6 +141,22 @@ class Orchestra(
 
     private fun assertCommand(command: AssertCommand) {
         command.visible?.let { assertVisible(it) }
+        command.notVisible?.let { assertNotVisible(it) }
+    }
+
+    private fun assertNotVisible(selector: ElementSelector) {
+        try {
+            assertVisible(selector)
```

Comment:
With this implementation:

* If the view does not exist at the time of calling the method, we will wait for assertVisible to time out before returning
* If the view does exist at the time of calling the method, we will throw an assertion failure immediately

I believe we want the inverse behavior

Reply:
Oh snap, yes you are right

Reply:
Addressed that. Now the caveat is - we will wait _either way_ whenever view is visible or not. The rationale is:

- If view is not currently visible, what if it will show up soon?
- If view is currently visible, what if it will disappear soon?

Reply:
This is going to introduce a ton of lag in almost every interaction won't it?

Reply:
Reduced waiting time in case if view is currently not visible to 2 seconds